go.mod

```diff
@@ -2,7 +2,4 @@ module github.com/pelletier/go-toml/v2
 
 go 1.15
 
-require (
-	github.com/stretchr/testify v1.7.0
-	golang.org/x/perf v0.0.0-20210220033136-40a54f11e909 // indirect
```

Comment:
not sure fi there is a reason for keeping this?

Reply:
not sure why it was added in the first place

Reply:
i try to limit the number of dependencies of go-toml. used to have 0 on v1, but i made an exception for testify on v2.